docs/semgrep-appsec-platform/jira.md

```diff
@@ -15,10 +15,22 @@ The Semgrep Jira integration allows you to create Jira tickets based on your Sem
 
 ## Prerequisites
 
-* This feature is in a **private beta**. To request access:
-  1. Fill out the [Request access to the Semgrep Jira integration closed beta](https://get.semgrep.dev/Jira-asana-linear-private-beta.html) form.
-  2. Contact your Technical Account Manager or Account Executive and let them know you want to try the Jira integration.
-* You must have a **Jira Cloud** plan. Jira Data Center (self-managed or on-premise) is not supported.
+- The Jira integration is in **public beta** for all existing Semgrep AppSec Platform users.
+- You must have a **Jira Cloud** plan. Jira Data Center (self-managed or on-premise) is not supported.
+- You must have at least one Jira project, which is where the tickets will be created.
+
+## Features
+
+The Semgrep Jira integration provides the following capabilities:
+
+- You can create tickets for findings from Semgrep Code, Supply Chain, and Secrets.
+- You can create a single ticket for multiple findings (up to 50) that were detected by a single rule, or create individual tickets per finding.
+- These tickets can be created in **multiple Jira projects**.
```

Comment:
```suggestion
- Tickets can be created in **multiple Jira projects** if manually specified at ticket creation time.
```

The way this is written right now might make it seem like you can do automatic project mapping (automatically choosing a Jira project based on the Semgrep project that the finding is in), which is not supported